tools/serverpod_cli/lib/src/util/locate_modules.dart

```diff
@@ -25,7 +25,8 @@ Future<List<ModuleConfig>?> locateModules({
           continue;
         }
 
-        if (!packageName.endsWith(_serverSuffix)) {
+        if (!packageName.endsWith(_serverSuffix) &&
+            packageName != 'serverpod') {
           continue;
         }
```

Comment:
Question: I don't understand what we are filtering away here?

Reply:
Since all serverpod packages end with the `_serverSuffix` we try and filter away everything that is not a serverpod package.